kovri.md

```diff
@@ -17,17 +17,35 @@ Kovri is a free, decentralized, anonymity technology based on I2P's open specifi
+
 
 ## 4) Examples
 
-Suppose Alice wants to send a transaction to Bob, and she begins the transaction.
-In order for her transaction to have the greatest chance of being included in the next block, she needs to reach as many nodes as possible.
-Alice connects to five nodes and these nodes then forward the transaction to other nodes, which in turn rebroadcast it repeatedly until many (or all) nodes in the network have this transaction.
-Unfortunately, Alice takes a risk when broadcasting her transaction.
-Some nodes may be logging IP addresses when transaction requests are received.
-Even though wallet addresses and the amount of Monero remain private, an attacker with enough resources could attempt to associate transactions with IP addresses to determine where transactions originate from.
-This could potentially lead to an attacker knocking on Alice's door or blocking her transaction from the rest of the network.
+Let's go through a few scenarios to see how this lightweight, security-focused software helps strengthen the unique privacy properties of Monero.
+
+Suppose Alice wants to send Monero to Bob.
+Alice's wallet creates a transaction, which then asks her daemon (her node) to broadcast it to the Monero network.
+Alice's node is connected to several nodes run by network peers, so those peers are first to receive a broadcast of her transaction.
+Those peers have peers of their own, to which they rebroadcast Alice's transaction.
+This process repeats until most - or all - nodes on the Monero network have Alice's transaction.
+
+In order for this relaying of transactions to occur, nodes must be able to communicate with one another.
+Nodes communicate by directing messages to the intended recipients using the recipient's IP address.
+Nearly all communications between computers are accomplished the same way.
+This means that it is possible to geographically trace data as it travels the internet, from start to finish and everywhere in between.
+
+Back to Alice, we now see that she is taking a risk by broadcasting her transaction.
+Some nodes may be logging IP addresses when transactions are received.
+Every one of Alice's node's network peers knows her IP address; therefore, each one of those peers has the ability to keep track of when a new transaction was first broadcast to them by Alice's node.
+
+Since Monero nodes tend to connect to only a limited number of peers, a transaction broadcast by Alice's node is not a conclusive indicator that Alice has initiated a transaction.
+And, even though the sender's and recipient's wallet addresses - as well as the amount of Monero sent - remain private, an adversary with enough resources could attempt to associate transactions with IP addresses to determine from where transactions originate.
+This could potentially lead to an adversary not relaying her transactions to the rest of the network; or - worse - arriving at her front door.
+
+What can Alice do to mitigate these threats? 
```

Comment:
Would repeat with line 63

Reply:
Oh, whoopsie.  You're right.  45 and 46 should go away.  And then I think 47 and 48 are slight improvements to 64.